files/runScanner.sh

```diff
@@ -1,18 +1,27 @@
 #!/bin/bash
-echo "setting up user & logfile:"
-if [[ -z ${UID} ]]; then
```

Comment:
I might need to test this here, currently just reading the code, but how do you ensure that the UID of the external folder mounted to /scans within the container fits the UID of the user running the brother script, to ensure the scan can actually be written into the folder? Especially in the case where the script is launched via the buttons on the scanner on not the web interface.